src/css/layout/table.less

```diff
@@ -21,7 +21,9 @@
   font-weight: normal;
   border: none;
   background-color: @table-header-color;
-
+  &.table-column-sorted{
```

Comment:
How about calling this class simply `highlight-column`? We are already in the `table` domain, so why repeat it. Also, `sorted` does not really reflect well what the class does, imho.

Either way, it needs a space before the curly ;)